src/path_rewriting.rs

```diff
@@ -337,7 +338,7 @@ pub fn rewrite_paths(
 
             // Try mapping a partial path to a full path.
             let rel_path =
-                if map_partial_path_needed && check_extension(&rel_path, PARTIAL_PATH_EXTENSION) {
+                if map_partial_path_needed && (check_extension(&rel_path, PARTIAL_PATH_EXTENSION) || check_extension(&rel_path, "kt")) {
```

Comment:
We should make PARTIAL_PATH_EXTENSIONS a vector with both "java" and "kt" and then update the code accordingly